rn-cli.config.js

```diff
@@ -0,0 +1,18 @@
+const defaultConfig = require('react-native/local-cli/core/default.config.js');
+
```

Comment:
This errors out: Error: Cannot find module 'react-native/local-cli/core/default.config.js'

Reply:
@machour It's my fault. RN 0.48+ is hard to be installed in China due to network issues, so I usually install 0.44.3. I will update it to suit our RN version.